.taskcluster.yml

```diff
@@ -0,0 +1,30 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.closed
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+          - push
+          - release
+    payload:
+      maxRunTime: 3600
+      image: node
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} repo && cd repo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} &&
+          yarn && yarn lint && yarn test && yarn build
```

Comment:
This would be clearer with each command on it's own line. Maybe even
```yaml
command:
  - /bin/bash
  - '--login'
  - '-c'
  - |
    set -e
    git clone {{...}}
    ...
    yarn build
```